### PR TITLE
New screenshots

### PR DESCRIPTION
**Pull request by CAnorbo**
+ Meta data fixes
+ Social images added

### CODE REVIEW

**Review by luzius1089 — COMMENTED** [review]
image

File: wwwroot/index.html [hunk 1]
```diff
@@ -338,14 +338,21 @@ <h2 class="mb-5">Frequently Asked Questions</h2>
...
+                            </div>
+
+                            <div class="box mt-4" id="addtowasabi">
                                 <h4 class="mb-4">How to add Ginger coordinator to Wasabi Wallet?</h4>
                                 <ol class="text-start">
-                                    <li>Make sure that you are using the latest Wasabi Wallet version.</li>
-                                    <li>Go to Settings -> Bitcoin tab -> Coordinator URI</li>
-                                    <li>Change https://api.wasabiwallet.io/ -> https://api.gingerwallet.io/</li>
+                                    <li>Make sure that you are using Wasabi Wallet 2.0.7.2 or earlier version.</li>
+                                    <li>Click to Search bar » Select Data folder » Find Config.json</li>
```
Comment (soosr) [inline wwwroot/index.html:354]: Click to Search bar » Type "Config File" » Press ENTER

File: wwwroot/index.html [hunk 2]
```diff
@@ -338,14 +338,21 @@ <h2 class="mb-5">Frequently Asked Questions</h2>
...
+                                <strong>https://api.gingerwallet.io/</strong>
+                            </div>
+
+                            <div class="box mt-4" id="addtowasabi">
                                 <h4 class="mb-4">How to add Ginger coordinator to Wasabi Wallet?</h4>
                                 <ol class="text-start">
-                                    <li>Make sure that you are using the latest Wasabi Wallet version.</li>
-                                    <li>Go to Settings -> Bitcoin tab -> Coordinator URI</li>
-                                    <li>Change https://api.wasabiwallet.io/ -> https://api.gingerwallet.io/</li>
+                                    <li>Make sure that you are using Wasabi Wallet 2.0.7.2 or earlier version.</li>
```
Comment (soosr) [inline wwwroot/index.html:353]: Changing the coordinator in the config file, works with all the versions. So probably this line can be removed.
Chaing the coordinator in the software is only avalable in and after 2.0.8.0.

Comment (CAnorbo) [reply wwwroot/index.html:353]: But after 2.0.7.2 fees will be 0%. That's why we added it.

File: wwwroot/index.html (hunk 1 above)
Comment (CAnorbo) [reply wwwroot/index.html]: Done.

File: wwwroot/index.html (hunk 2 above)
Comment (soosr) [reply wwwroot/index.html:353]: I don't think so, they won't yet add it in 2.0.8.0 either. And I am not even sure if they even add it later.

**Comment by luzius1089** [comment]
Our version will be v2.0.8 as well, won't be?

**Comment by soosr** [comment]
> Our version will be v2.0.8 as well, won't be?

Wasabi is currently on 2.0.8.0 and since we placed our changes on top of theirs I think we should be on 2.0.8.1

**Comment by CAnorbo** [comment]
We have 2.0.8.1 on the website rn.

**Review by luzius1089 — APPROVED** [review]

**Pull request merged**